addons/mail/static/src/scss/emojis.scss

```diff
@@ -13,11 +13,19 @@
     }
 }
 
+.o_form_view {
+    // Widget(emojis) should be hidden when field has "invisible modifier" !
+    .o_invisible_modifier + .o_mail_add_emoji{
```

Comment:
Sugg:

```
// Emojis widgets should hide the emoji dropdown button when the field is invisible.
// This is necessary because the button is added *after* the main element (and not inside)
// (see '_attachEmojisDropdown' for more details)
```